docs/src/contribute/pull-requests.md

```diff
@@ -116,8 +114,6 @@ If there are any failing tests, update your code until all tests pass.
 
 With your code ready to go, this is a good time to double-check your submission to make sure it follows our conventions. Here are the things to check:
 
-* Make sure your commit is formatted correctly.
-* The pull request must have a description. The description should explain what you did and how its effects can be seen.
```

Comment:
A pull request hasn't been created yet, so I guess this fits better in Step 7 down below.